### ISSUE TITLE
Create a command to import a single po file

### ISSUE THREAD

**Issue by pascalchevrel** [mozilla-l10n/langchecker#102]
We have all the code we need to import po files from locamotion but we need a command to import strings from a single .po file which can we useful at time (on conributor sent me over the week end the Australis strinfs as po files).

a command like:
poimport path/to/po/file path/to/lang/file 

With a third optional flag, 'replace', in case we want to replace our existing translations in the lang file.

Assigning to Theo, will commit my script to activate lang files in parallel (separate issue) so as to give him a better example of a command line php script than lang_update
